sw/tools/generators/gen_periodic.ml

```diff
@@ -109,7 +109,7 @@ let write_settings = fun xml_file out_set telemetry_xml ->
   fprintf out_set "<!-- This file has been generated by gen_periodic from %s -->\n" xml_file;
   fprintf out_set "<!-- Version %s -->\n" (Env.get_paparazzi_version ());
   fprintf out_set "<!-- Please DO NOT EDIT -->\n\n";
-  fprintf out_set "<settings>\n";
+  fprintf out_set "<settings target=\"ap|nps\">\n";
```

Comment:
See the general comment ;) this is btw true, since there is no telemetry in fbw.